src/FsAutoComplete.Core/AdaptiveExtensions.fs

```diff
@@ -575,10 +599,7 @@ module AsyncAVal =
             let ref =
               RefCountingTaskCreator(
                 cancellableTask {
-                  let! ct = CancellableTask.getCancellationToken ()
-                  let it = input.GetValue t
-                  use _s = ct.Register(fun () -> it.Cancel())
-                  let! i = it
```

Comment:
Most of this stuff is happening with the extension methods that use https://github.com/fsharp/FsAutoComplete/blob/9494bbf9554844dde4dcb740cd2839c6d507e414/src/FsAutoComplete.Core/AdaptiveExtensions.fs#L403-L408